bundler/lib/dependabot/bundler/native_helpers.rb

```diff
@@ -1,10 +1,36 @@
 # frozen_string_literal: true
 
+require "dependabot/shared_helpers"
+
 module Dependabot
   module Bundler
     module NativeHelpers
-      def self.helper_path
-        "bundle exec ruby #{File.join(native_helpers_root, 'run.rb')}"
+      def self.run_bundler_subprocess(function:, args:, bundler_version:)
+        bundler_env_version = bundler_version.tr("v", "")
```

Comment:
This feels a bit funky, might be easier to do the version detection here instead